frontend/staff-dashboard/src/pages/discounts/edit.tsx

```diff
@@ -0,0 +1,42 @@
+import { 
+    useForm,
+    Form,
+    Input,
+    Select,
+    Edit,
+ } from "@pankod/refine-antd";
+
+import { IDiscount } from "interfaces";
+
+export const DiscountEdit = () => {
+    const { formProps, saveButtonProps, queryResult } = useForm<IDiscount>();
+
+    return (
+        <div>
+            <Edit saveButtonProps={saveButtonProps}>
+                <Form {...formProps} layout="vertical">
+                    <Form.Item label="Discount Code" name="discount_code">
+                        <Input />
+                    </Form.Item>
+                    <Form.Item label="Redemption Type" name="redemption_type">
+                        <Select options={[
+                            { label: 'Unlimited', value: 'unlimited' },
+                            { label: 'One-Time', value: 'one-time' },
+                            { label: 'One Time Per User', value: 'one-time-per-user' }, 
+                        ]}></Select>
+                    </Form.Item>
+                    <Form.Item label="Discount Type" name="discount_type">
+                        <Select options={[
+                            { label: 'Percent Off', value: 'percent-off' },
+                            { label: 'Dollars Off', value: 'dollars-off' },
+                            { label: 'Fixed Price', value: 'fixed-price' }, 
+                        ]}></Select>
+                    </Form.Item>
+                    <Form.Item label="Amount" name="amount">
+                        <Input />
```

Comment:
Is there a way to adjust the precision on this to only show 2 digits after the decimal? Even better if there's a currency input you can utilize for both this and the create form.

Reply:
There was a InputNumber component for this that allows for a set precision - I swapped these over (on create and edit) to use that. (It has a bit of a styling bug, as the field isn't full width like the rest, but since there'll need to be a general styling overhaul here later I'm ignoring it for now.)